locale/fr.po

```diff
@@ -24,34 +24,35 @@ msgstr "Impossible de charger le fichier de préférences de l'extension"
 
 #: prefs.js:317
 msgid "WebCam Caps"
-msgstr ""
+msgstr "Capabilités de la webcam"
 
 #: prefs.js:732
-#, fuzzy
 msgid "No Gstreamer pipeline found"
 msgstr "Pipeline GStreamer"
 
 #: prefs.js:766
 msgid "Version: "
-msgstr ""
+msgstr "Version : "
```

Comment:
Is the space between "Version" and ":" on purpose?

Reply:
Yes absolutely, in french we have a space between the words and the colon. See this for instance http://www.oph-montigny.fr/UserFiles/File/formulaire-de-demande.jpg